packages/amplify_auth_cognito/test/amplify_auth_cognito_stream_controller_test.dart

```diff
@@ -49,12 +48,12 @@ void main() {
     );
   }
 
-  test('Amplify.Hub.listen with auth channel returns a StreamSubscription', () async {
-    await Amplify.addPlugin(auth);
-    var sub = Amplify.Hub.listen([HubChannel.Auth], (msg) {});
-    expect(sub, isInstanceOf<StreamSubscription>());
-    await sub.cancel();
-  });
+  // test('Amplify.Hub.listen with auth channel returns a StreamSubscription', () async {
+  //   await Amplify.addPlugin(auth);
+  //   var sub = Amplify.Hub.listen([HubChannel.Auth], (msg) {});
+  //   expect(sub, isInstanceOf<StreamSubscription>());
+  //   await sub.cancel();
+  // });
```

Comment:
is this a todo?

Reply:
These are unneeded in this file - they are testing basic hub functionality, and removing them allows us to get rid of the amplify_flutter dependency. Removing this commented code.